src/main/sensors/esc_sensor.c

```diff
@@ -1106,21 +1008,30 @@ static void kontronikSensorProcess(timeUs_t currentTimeUs)
             uint32_t crc = kontronikDecodeCRC(kontronikPacketLength - KON_CRC_LENGTH);
             if (calculateCRC32(buffer, kontronikPacketLength - kontronikCrcExclude - KON_CRC_LENGTH) == crc) {
                 uint32_t rpm = buffer[7] << 24 | buffer[6] << 16 | buffer[5] << 8 | buffer[4];
+                int16_t  throttle = (int8_t)buffer[24];
```

Comment:
Can this be negative? Why the (int8_t)?

Reply:
Possibly - docs say ...
 *     24:      Throttle % (-100..100)
 ie apply offset for 0-200, then scale *5 to be equivalent to .pwm 0..100 (*10)
     escSensorData[0].throttle = (throttle + 100) * 5;
     escSensorData[0].pwm = pwm * 10;